### ISSUE TITLE
Depth of Orderbook and Sequence Numbers

### ISSUE THREAD

**Issue by pwnzya** [JKorf/Bittrex.Net#174]
Hi, thanks for the quick update to BittrexSymbolOrderBook class etc.
About the sequence number provided in GET request's ResponseHeaders..., but of course!   **Face palm to myself**

I think the following changes are still required:
- V3 REST API method GetOrderBook and GetOrderBookAsync have the depth parameter.
- Websocket API method SubscribeToOrderBookUpdatesAsync's depth parameter validation needs to allow 500 as its value, not 100.

Quote from the [documentation](https://bittrex.github.io/api/v3#operation--markets--marketSymbol--orderbook-get), 
> depth: integer (int32) in query
> maximum depth of order book to return (optional, allowed values are [1, 25, 500], default is 25)

Finally, in the updated class BittrexSymbolOrderBook, calls to GetOrderBookAsync method need to specify the depth of orderbook because the sequence numbers are different depending on the depth.
Quote from the [documentation](https://bittrex.github.io/api/v3#method-Orderbook):
> Note: You must get the orderbook snapshot from the same depth as you are subscribed to on the websocket. Sequence numbers are not the same for different depths.

Speaking of the sequence numbers, the following methods also need to get the sequence number from the ResponseHeaders:

- GetBalances
- GetOpenDeposits
- GetSymbolSummaries
- GetTickers
- GetSymbolTrade
- GetSymbolKlinesRecent
- GetOpenOrders

**Comment by JKorf** [comment]
You're right, I'm fixing the points you mentioned. 

It is kind of annoying to add the sequence numbers though, since these endpoints all (except for the order book) return lists of results, so there isn't an object I can just slap the Sequence property on, unlike how I did with the order book. 
So they would all need wrappers to allow the Sequence property to be returned in the result.

Alternatively, I already made a GetSequence extension method for the response headers, which might be good enough: 
````C#
var x = client.GetBalances();
var sequence = x.ResponseHeaders.GetSequence();
````

What are your thoughts on this?
